topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -142,12 +142,12 @@ Sequence quality control is therefore an essential first step in your analysis.
 
 > ### {% icon hands_on %} Hands-on: Quality control
 >
-> 1. **FastQC** {% icon tool %} with the following parameters:
+> 1. [**FastQC**](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) {% icon tool %} with the following parameters:
```

Comment:
You need for all links to remove the portion until the `=` to keep only the part starting with `toolshed`

Reply:
Here's the full [guide on how to do it](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links) if it helps :)

Reply:
Thank you so much :) 
When I did this like the example in (https://github.com/galaxyproject/training-material/tree/main/topics/transcriptomics/tutorials/scrna-plant), I was not able to test it with the local galaxy since the links where not found, however, I will do it as the guide

Reply:
Yeah, that's expected. They're not real links, they're not expected to be clickable. We just re-use the link syntax since it's similar in concept and people were familiar with that syntax. It's the `{% tool %}` portion that does the magic.

If you need to test locally it's a bit complicated, you need to setup a proxy that proxies both your galaxy + the GTN. There's a very short note at the end of https://docs.galaxyproject.org/en/latest/admin/special_topics/gtn.html, but I wouldn't recommend going through the trouble of testing that locally.

Reply:
Thank you so much for the great explanation :)